src/engine/UNSATCertificate.cpp

```diff
@@ -0,0 +1,577 @@
+/*********************                                                        */
+/*! \file UNSATCertificate.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Omri Isac, Guy Katz
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2022 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+ **/
+
+#include "UNSATCertificate.h"
+
+CertificateNode::CertificateNode( Vector<Vector<double>> *initialTableau, Vector<double> &groundUpperBounds, Vector<double> &groundLowerBounds )
+    : _children()
```

Comment:
empty constructors dont need to be called explicitly. Remove _children(), _headSplit(), etc